src/frontend/src/components/headerComponent/index.tsx

```diff
@@ -179,6 +185,22 @@ export default function Header(): JSX.Element {
               <IconComponent name="MoonIcon" className="side-bar-button-size" />
             )}
           </button>
+
+          <button
```

Comment:
Could you please add a tooltip to explain the function of this button?

Reply:
Should I use shadTooltipComponent or something else?